pallets/tendermint-light-client/src/lib.rs

```diff
@@ -0,0 +1,337 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+//! This pallet is an on-chain light-client for tendermint (Cosmos) based chains
+//! It verifies headers submitted to it via on-chain transactions, performed by a so-called relayer
+//! It is a part of the Aleph Zero <-> Terra bridge
```

Comment:
Not sure if we should mention Terra explicitly here since this client should be useable by any Tendermint-based chain.

Reply:
done